packages/devextreme/js/__internal/ui/m_file_uploader.ts

```diff
@@ -834,19 +844,21 @@ class FileUploader extends Editor<FileUploaderProperties> {
       disabled: this.option('readOnly'),
       hoverStateEnabled: this.option('hoverStateEnabled'),
     });
-    this._selectFileDialogHandler = this._selectButtonClickHandler.bind(this);
 
     // NOTE: click triggering on input 'file' works correctly only in native click handler when device is used
     if (devices.real().deviceType === 'desktop') {
-      this._selectButton.option('onClick', this._selectFileDialogHandler);
+      this._selectButton.option('onClick', this._selectFileDialogClickHandler);
```

Comment:
[mistake?]
How tests can be green if context is lost here? You should use arrow function or bind here to keep it working with `this.` if i miss nothing

Reply:
c1058a4c403d37dcd8e1337200ce9985c3463039